### DISCUSSION TITLE
Question about a mod conflict

### DISCUSSION THREAD

**Discussion by StrikerMercury** [xivdev/Penumbra#178] [Q&A]
Hey, is this the right spot to ask for help with a Penumbra mod conflict? I am brand new to it and don't know where else to turn for assistance, Google did not help. I have no other mods installed other than The Body SE and have the settings done right, but when I enable it I get three conflicts pop us that read this:

chara/human/c0101/obj/body/b0001/texture/--c0101b0001_d.tex
chara/human/c0101/obj/body/b0001/texture/--c0101b0001_n.tex
chara/human/c0101/obj/body/b0001/texture/--c0101b0001_s.tex

**Comment by StrikerMercury** [comment]
Answered elsewhere!

**Reply by Slorch** [reply]
What was the answer? I'm having a similar issue but cant find much information on it.

**Reply by lhsaffar** [reply]
"Self conflicts" inside a mod MOST of the time are a packaging 'error' from the modder because Textools would imply overwrite the duplicate files and not ask questions about it. Penumbra does not work the same way, but 99% of the time, self-conflicts are NOT an issue.

TLDR if the mod functions as intended, self-conflicts mean nothing except that the packaging for this mod was made without Penumbra in mind.
